tests/contracts/TestRulesFactory.sol

```diff
@@ -26,32 +27,25 @@ contract TestRulesFactory is Stakable, ValidationRulesStorage {
         // note that the 'EXT*'/'CALL*' opcodes are allowed on the zero code address if it is later deployed
         address create2address = getAddress(nonce, _entryPoint);
         if (rule.eq("EXTCODEx_CALLx_undeployed_sender")) {
-            // CALL
-            create2address.call("");
-            // CALLCODE
-            assembly {
-                let res := callcode(5000, create2address, 0, 0, 0, 0, 0)
-            }
-            // DELEGATECALL
-            create2address.delegatecall("");
-            // STATICCALL
-            create2address.staticcall("");
-            // EXTCODESIZE
-            emit Uint(create2address.code.length);
-            // EXTCODEHASH
-            emit Uint(uint256(create2address.codehash));
-            // EXTCODECOPY
-            assembly {
-                extcodecopy(create2address, 0, 0, 2)
-            }
+            ValidationRules.runFactorySpecificRule(nonce, rule, _entryPoint, create2address);
+        }
+        else if (rule.eq("DELEGATECALL:>EXTCODEx_CALLx_undeployed_sender")) {
```

Comment:
do you really think this name is meaningful?
also, slicing such a long name, and then passing the rest elsewhere is unmanageable.
maybe if you check for prefix, remove it and forward from there,